core/src/main/scala/com/microsoft/azure/synapse/ml/logging/common/CommonUtils.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.logging.common
+
+import com.microsoft.azure.synapse.ml.logging.common.WebUtils._
+import scala.util.{Success, Try}
+import spray.json.{JsValue}
+import spray.json.DefaultJsonProtocol.IntJsonFormat
+import spray.json.DefaultJsonProtocol.StringJsonFormat
+
+object CommonUtils {
```

Comment:
maybe just group this with Web utils to reduce file cluitter

Reply:
Alright.